.travis.yml

```diff
@@ -12,7 +12,8 @@ env:
   - MONGOID_VERSION=4
   - MONGOID_VERSION=5
   - MONGOID_VERSION=6
-  - MONGOID_VERSION=7
+  - MONGOID_VERSION=70
```

Comment:
If I were to nitpick, we can just do exact things here, e.g. `7.0` and `7.1`, then in Gemfile `when '7.1'` instead of the regex. Seems unnecessarily complicated.